src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -208,29 +213,33 @@ class LazyMobXTableStore<T> {
     @computed public get downloadData()
```

Comment:
I think the current solution, which @adamabeshouse summarized well, could be the default behavior. Perhaps other (future) download formats could be implemented by overriding this default function? So by the time we need more specific download formats for the patient view, we could have a new type, e.g. PatientViewTable, which extends LazyMobXTable with its own extra download formats. Is such a scenario feasible with the current code structure?